development/sentry-publish.js

```diff
@@ -82,11 +82,13 @@ async function start() {
     ]);
   }
 
-  let distDirectory = 'dist';
+  let distDirectory = 'build-dist-browserify/dist';
```

Comment:
Hmm. Not ideal. Previously when we used `dist` here, it worked equally well locally as well as on CI. CI would move the build to `dist` before uploading sourcemaps.

Now with this non-default hard-coded build directory, this will no longer work for local builds unless you happen to know how this works internally. It also creates a dependence between the build/directory naming and this script that may be easy to miss.

Thoughts on passing in the build directory as a parameter instead?